pygbx/gbx.py

```diff
@@ -135,6 +149,11 @@ def __init__(self, obj):
         bp = ByteReader(self.data)
         self._read_node(self.class_id, -1, bp)
 
+    """Closes the current file connection"""
+
+    def close(self):
```

Comment:
Also note that `self.f` may not even exist when calling this method when constructing a Gbx instance with raw byte data. This should be handled in some way, either doing nothing or throwing an exception.